third_party/txt/src/txt/font_asset_provider.h

```diff
@@ -28,6 +28,9 @@ class FontAssetProvider {
   virtual size_t GetFamilyCount() const = 0;
   virtual std::string GetFamilyName(int index) const = 0;
   virtual SkFontStyleSet* MatchFamily(const std::string& family_name) = 0;
+
+ protected:
+  static std::string CanonicalFamilyName(std::string family_name);
```

Comment:
Where is this method implemented? I don't seem to see it anywhere.

Reply:
Forgot to add that file - PTAL